index.md

```diff
@@ -49,3 +49,10 @@ the satellite data of the Copernicus program, but who do not want to work with
 huge amounts of scientific data directly. We will provide examples and
 libraries to quickly get you started without being an expert in satellite data
 analysis.
+
+Publishments
```

Comment:
I would call it 'Presentations' we would then later link other presentations we will have held at conferences.